web_console/web_terminal/troubleshooting-web-terminal.adoc

```diff
@@ -9,6 +9,44 @@ toc::[]
 [id="troubleshooting-web-terminal-network-policies"]
 == Web terminal and network policies
 
-The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state.
+The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state until a `context deadline exceeded error` occurs.
```

Comment:
IBM Style word usage advises against using the terms _launch_ and _appears_. Consider a revision like this one:
```suggestion
The web terminal might fail to start if the cluster has network policies configured. To start a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to start and the terminal panel is in a loading state until a `context deadline exceeded error` occurs.
```

Reply:
Thank you, the suggestion looks great, I've applied it in my latest force push